manage.py

```diff
@@ -64,6 +64,13 @@ def setup_dev():
     """Runs the set-up needed for local development."""
     setup_general()
 
+    admin_email = 'wvr@gmail.com'
+    if User.query.filter_by(email=admin_email).first() is None:
```

Comment:
@jondubin what he currently has is better than `if not User.query.filter_by(email=email).first():`

From PEP8:

>Comparisons to singletons like None should always be done with 'is' or 'is not', never the equality operators.

>Also, beware of writing "if x" when you really mean "if x is not None" -- e.g. when testing whether a variable or argument that defaults to None was set to some other value. The other value might have a type (such as a container) that could be false in a boolean context!